tests/test_geovector.py

```diff
@@ -169,11 +169,11 @@ def test_reproject_respects_units():
     # Obtained from geojson.io
     expected_gv = GeoVector(
         Polygon([
-            [-103.9206624695924, 31.47120806441344],
-            [-103.92458240704434, 31.52607263749016],
-            [-103.88935335998022, 31.527915387915815],
-            [-103.88545482875507, 31.4730496225029],
-            [-103.9206624695924, 31.47120806441344]
+            [-103.9201941683759, 31.47107738840808],
```

Comment:
I tried updating this test because it failed in one of the jobs, but now that I changed it, it failed in Python 3.5. Not sure what to do here, apart from relaxing the tolerance.

Reply:
Probably pyproj 3 fault. Following @tomaslink advice and limiting to `pyproj<3`, for now.